e2e_testing/xfail_sets.py

```diff
@@ -263,6 +263,9 @@
     # ERROR: Exception: Unsupported: return type List[Tensor] in schema for aten.unbind.int
     "UnbindIntListUnpack_Module_basic",
     "UnbindIntGetItem_Module_basic",
+
+    "ScatterValueFloatModule_basic",
```

Comment:
What's the failure error? This list is organized based on the error message. If the error is a new one, can you add it as a comment?